processor-line.cpp

```diff
@@ -18,6 +18,5 @@ geometry_processor::wkbs_t
 processor_line::process_relation(osmium::Relation const &,
                                  osmium::memory::Buffer const &ways)
 {
-    // XXX are multilines really acceptable?
-    return m_builder.get_wkb_multiline(ways, false);
+    return m_builder.get_wkb_multiline(ways, 1000000);
```

Comment:
Given the size of North American road relations and the Mercator scaling, it's safer to use 10 000 000.